Freezing on installation stage 'Unpacking st2common ...'

I've played a bit, spinned up the new AWS instance with [Ubuntu 14 LTS](https://aws.amazon.com/marketplace/pp/B00JV9JBDS).

Steps I did: 
```sh
sudo sh -c "wget -qO- https://get.docker.io/gpg | apt-key add -"
sudo sh -c "echo deb http://get.docker.io/ubuntu docker main > /etc/apt/sources.list.d/docker.list"
sudo apt-get update
sudo apt-get -y install lxc-docker
sudo apt-get -y install python-pip git
pip install docker-compose
git clone https://github.com/dennybaa/st2-packages.git
cd st2-packages

docker-compose run debian
```

My problem it just stucks here: 
```log
Bundle complete! 2 Gemfile dependencies, 16 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.
Warning: Permanently added 'iwheezy,172.17.0.2' (ECDSA) to the list of known hosts.
DEBUG: scripts/install.sh has been invoked!
/root/build/tmp.IiYwNAjIgR/st2common_0.14dev-1_amd64.deb

===> Staring installation of package: st2common_0.14dev-1_amd64
Selecting previously unselected package st2common.
(Reading database ... 26168 files and directories currently installed.)
Unpacking st2common (from .../st2common_0.14dev-1_amd64.deb) ...
```

Waited for 20 minutes or so on c4.large instance. It just freezed with no further output. Tried 2 times on new machines.

----

Some configuration I'm missing maybe (using all defaults)?

Ah, had no luck running it in Vagrant `ubuntu/trusty64` thus far. This time at build stage:

```sh
# sudo docker-compose run debian

===> Starting build on bwheezy
     package list: [st2common st2actions st2api st2auth st2client st2debug st2exporter st2reactor st2bundle]
Warning: Permanently added 'bwheezy,172.17.0.4' (ECDSA) to the list of known hosts.
/tmp/tmp.NQAcf99WDZ ~
/tmp/tmp.NQAcf99WDZ/st2common /tmp/tmp.NQAcf99WDZ ~

===> Starting package st2common build
# populate version should be run before any pip/setup.py works
sh ../scripts/populate-version.sh
touch .stamp-populate_version
debchange -v 0.14dev-1 -M "automated build version: 0.14dev"
dpkg-buildpackage: source package st2common
dpkg-buildpackage: source version 0.14dev-1
dpkg-buildpackage: source changed by unknown <opsadmin@stackstorm.com>
 dpkg-source --before-build st2common
dpkg-buildpackage: host architecture amd64
 debian/rules clean
dh clean --with python-virtualenv
   dh_testdir
   dh_clean
 debian/rules build
dh build --with python-virtualenv
   dh_testdir
   dh_auto_configure
 debian/rules binary
dh binary --with python-virtualenv
   dh_testroot
   dh_prep
   debian/rules override_dh_installdirs
make[1]: Entering directory `/tmp/tmp.NQAcf99WDZ/st2common'
dh_installdirs
/tmp/tmp.NQAcf99WDZ/st2common/debian/st2common.dirs: 1: /tmp/tmp.NQAcf99WDZ/st2common/debian/st2common.dirs: /etc/st2: not found
/tmp/tmp.NQAcf99WDZ/st2common/debian/st2common.dirs: 2: /tmp/tmp.NQAcf99WDZ/st2common/debian/st2common.dirs: /etc/logrotate.d: Permission denied
/tmp/tmp.NQAcf99WDZ/st2common/debian/st2common.dirs: 3: /tmp/tmp.NQAcf99WDZ/st2common/debian/st2common.dirs: /var/log/st2: not found
make[1]: Leaving directory `/tmp/tmp.NQAcf99WDZ/st2common'
dh_installdirs: problem reading debian/st2common.dirs:
make[1]: *** [override_dh_installdirs] Error 127
make: *** [binary] Error 2
dpkg-buildpackage: error: debian/rules binary gave error exit status 2

===> Finished package st2common build sucessfully
/tmp/tmp.NQAcf99WDZ ~
cp: cannot stat `st2common*.deb': No such file or directory
```

--- a/README.md
+++ b/README.md
@@ -62,7 +62,7 @@ Let's describe a few important configuration variables:
 
 Issue the command bellow and the whole suite will be run, after it finishes you can find packages in your `/tmp/st2-packages` directory.
 ```
-docker-compose run ubuntu
+docker-compose up ubuntu
 ```
 
 ## License and Authors

